### ISSUE TITLE
test_array_constructor_noreturn crashes omniscidb server when CUDA enabled

### ISSUE THREAD

**Issue by pearu** [xnd-project/rbc#94]
```
NvidiaKernel.cpp:96 Check failed: cuLinkAddData_v2(link_state, CU_JIT_INPUT_PTX, static_cast<void*>(const_cast<char*>(ptx.c_str())), ptx.length() + 1, 0, 0, nullptr, nullptr) == CUDA_SUCCESS (218 == 0) 
```
`OmniSci Version: 5.4.0dev-20200604-fc4a06f9fc`

**Comment by pearu** [comment]
With https://github.com/omnisci/omniscidb-internal/pull/4714 we have
```
2020-08-10T16:35:03.361699 F 27368 10 NvidiaKernel.cpp:126 Check failed: cuLinkAddData_v2(link_state, CU_JIT_INPUT_PTX, static_cast<void*>(const_cast<char*>(ptx.c_str())), ptx.length() + 1, 0, 0, nullptr, nullptr) == CUDA_SUCCESS (218 == 0) ptxas application ptx input, line 562; error   : Call has wrong number of parameters
ptxas fatal   : Ptx assembly aborted due to errors
Aborted (core dumped)
```

**Comment by pearu** [comment]
Resolved via #209
